client/my-sites/plugins/plugin-details.jsx

```diff
@@ -142,14 +146,15 @@ function PluginDetails( props ) {
 	// Fetch WPorg plugin data if needed
 	useEffect( () => {
 		if ( isProductListFetched && ! isMarketplaceProduct && ! isWporgPluginFetched ) {
-			dispatch( wporgFetchPluginData( props.pluginSlug ) );
+			dispatch( wporgFetchPluginData( props.pluginSlug, translate.localeSlug ) );
```

Comment:
`translate.localeSlug` serves as a locale fallback for logged-out users where the URL `lang` parameter is used.